client/package.json

```diff
@@ -25,7 +25,7 @@
     "webpack": "5.75.0",
     "webpack-cli": "5.0.1"
   },
-  "proxy": "https://planz.example.com/",
+  "proxy": "https://planz.bcholmes.org",
```

Comment:
Can you restore this back to the example.com address?

Reply:
Sure. So every time a dev wants to login locally, they'll have to update this? I can add a clarifying comment to the component then.

Reply:
Yeah. We expect other devs to use different test servers. I think there's a note in the `client/README.md` about changing the proxy entry to run the local server.